src/layout/section/highlightSection/highlightSection.tsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react'
 import BaseSection, { SectionContentSize } from 'layout/section/baseSection'
+import SubHeader from 'subHeader'
 
 export interface HighlightSectionProps {
   readonly className?: Classcat.Class
   readonly contentClassName?: Classcat.Class
-  readonly children: JSX.Element | string
+  readonly children: React.ReactNode
+  readonly title?: string
 }
 
 /**
  * A specialized section with an highlighting background color.
  */
 const HighlightSection = (props: HighlightSectionProps) => {
-  const { className, children, contentClassName } = props
+  const { className, children, contentClassName, title } = props
   return (
     <BaseSection
+      tagName="article"
       className={className}
       contentClassName={contentClassName}
       contentSize={SectionContentSize.LARGE}
     >
+      {title !== null && <SubHeader>{title}</SubHeader>}
```

Comment:
What if title is `undefined`, use this instead:
`{title && <SubHeader>{title}</SubHeader>}`

Reply:
Done